Changes

```diff
@@ -308,6 +308,9 @@ OCaml 4.09.0
   (Damien Doligez, reports by John Whitington and Alain Frisch,
    review by Jeremy Yallop and Gabriel Scherer)
 
+- #8530: List.sort: avoid duplicate work by chop
+  (Guillaume Munch-Maccagnoni, review by David Allsopp and Damien Doligez)
+
```

Comment:
This is placed in the 4.09 section, but the PR will go in trunk, could you move the entry upwards in the trunk section?

Reply:
Fixed, thanks.